services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/Persistence.kt

```diff
@@ -211,6 +206,22 @@ class HankeEntity(
         }
     }
 
+    fun alkuPvm(): ZonedDateTime? {
+        return listOfHankeAlueet
+            .map { it.haittaAlkuPvm }
+            .filterNotNull()
```

Comment:
This can be simplified:
```suggestion
            .mapNotNull { it.haittaAlkuPvm }
```

Reply:
Done.